assets/i18n/en.json

```diff
@@ -616,13 +616,20 @@
     "I18N_FOOTER_BROWSE_LIBRARY": "Browse the Library",
     "I18N_FOOTER_CONTACT_US": "Contact Us",
     "I18N_FOOTER_CONTRIBUTE_ALL_CAPS": "CONTRIBUTE",
+    "I18N_FOOTER_COPYRIGHT": "© 2023 The Oppia Foundation",
```

Comment:
I think we should not hardcore the year, we can maybe add an angular logic to get the current year. What do you think?

Reply:
Indeed, that would help in code maintainability. I have modified it. PTAL